app/shared/availability-view/TimelineGroups/TimelineGroups.js

```diff
@@ -15,6 +16,16 @@ function scrollToInitial(element) {
   }
 }
 
+const scrollStickies = (function () {
+  function scroll(scrollContainer) {
+    const stickies = scrollContainer.querySelectorAll('.sticky');
+    stickies.forEach((sticky) => {
+      sticky.scrollLeft = scrollContainer.scrollLeft; // eslint-disable-line no-param-reassign
+    });
+  }
+  return throttle(scroll, 50);
```

Comment:
- [ ] Could a lower value be used here? 10ms for example? It would make the stickiness on sideways scrolling a lot smoother. Would it cause some other issues?